### DISCUSSION TITLE
[Note] Canadian Restricted Version of NBC(and US OTA Network)

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#192] [Show and tell]
encodercdn-ca Including Canadian Cable Channel (But Not of All).
So, It Means Including US OTA Channel. (NBC-CBS-FOX-abc)
Sample 1:
http://encodercdn1.frontline.ca/vantrix/ltv/Bell_NBCHB_AtopItci_2015_01_12_HD/hls_hd.m3u8
(Linked to NBC Buffalo, It Called Simply NBC in Canadian Cable)
Sample 2:
http://encodercdn1.frontline.ca/encoder183/output/NBC_Boston_720p/playlist.m3u8
(Linked to NBC Boston(WBTS-CD1), It Called NBC East in Canadian Cable)
Sample 3:
http://encodercdn1.frontline.ca/bespin/output/NBC_Seattle_720p/playlist.m3u8
(Linked to NBC Seattle(KING-TV1), It Called NBC West in Canadian Cable)

and Found US Channel Following
NBC - Buffalo,Boston,Seattle
CBS -  Buffalo,Seattle
FOX - Buffalo,Boston,Seattle
abc - Buffalo

If You Need Those, I can Add This Link.
But I Worried about "Simultaneous Substitution" (More Info: https://crtc.gc.ca/eng/television/publicit/america.htm)

Example:  "NBC's Late Night Talk Shows" are Airing on CTV2 in Canada, During This Program Like This. Sample Feed's Video Changes to CTV2." (As Results, Screen bug Change NBC to CTV2, It Makes Confusion to Users)

It Will Effective to Sample 1(Confimed) .
 "Sample 2" (Not Effective)
 "Sample 3" (Not Confimed).
Prof: (from Days our Lives)
(Sample1)
![71982BDA-606D-447E-A436-2754762ECE20](https://user-images.githubusercontent.com/86145713/146426460-7e457abc-b9bf-473c-a3ff-a8c4636e19b2.jpeg)
(Sample2)
![B6A3B421-DF7D-497C-B635-8CE5DB8A45B3](https://user-images.githubusercontent.com/86145713/146426820-e6b03beb-892c-4818-bbe3-536d10f73e1f.jpeg)

**Comment by BellezaEmporium** [comment]
1) Works for me
2) Works for me (take off _720p to have 480p version)
3) Works for me (take off _720p to have 480p version)

For all 3 of them, when advertisments stroke in, it was US adverts.

**Comment by chamby90** [comment]
@Central302 would you mind sharing the CBS and FOX links?

**Reply by chamby90** [reply]
For what its worth... I was able to figure out the Buffalo channels based on the URL string, but not having the same luck with the Boston or Seattle URLs

**Reply by chamby90** [reply]
but... i see you have put them in the m3u playlist.. so no worries, im good.  Thanks for adding those
